### PR TITLE
Contributions from the fd.io VPP project.

### PR DESCRIPTION
**Pull request by pvinci**

### CODE REVIEW

File: scapy/config.py [hunk 1]
```diff
@@ -613,7 +613,7 @@ class Conf(ConfClass):
     netcache = NetCache()
     geoip_city = None
     load_layers = ['bluetooth', 'bluetooth4LE', 'dhcp', 'dhcp6', 'dns',
-                   'dot11', 'dot15d4', 'eap', 'gprs', 'hsrp', 'inet',
+                   'dot11', 'dot15d4', 'eap', 'geneve', 'gprs', 'hsrp', 'inet',
```
Comment (gpotter2) [inline scapy/config.py]: `geneve` is a contrib module, and therefore shouldn't be loaded by default

**Comment by gpotter2** [comment]
Thanks for the PR !

You have a few PEP8 failures:

```
flake8 runtests: commands[0] | flake8 scapy/
scapy/layers/l2.py:429:20: E201 whitespace after '['
scapy/layers/l2.py:429:35: E231 missing whitespace after ','
scapy/layers/l2.py:429:37: E231 missing whitespace after ','
scapy/layers/l2.py:430:36: E231 missing whitespace after ','
scapy/layers/l2.py:430:38: E231 missing whitespace after ','
scapy/layers/l2.py:431:35: E231 missing whitespace after ','
scapy/layers/l2.py:431:37: E231 missing whitespace after ','
scapy/layers/l2.py:432:34: E231 missing whitespace after ','
scapy/layers/l2.py:432:36: E231 missing whitespace after ','
scapy/layers/l2.py:433:33: E231 missing whitespace after ','
scapy/layers/l2.py:433:35: E231 missing whitespace after ','
scapy/layers/l2.py:434:42: E231 missing whitespace after ','
scapy/layers/l2.py:434:44: E231 missing whitespace after ','
scapy/layers/l2.py:435:40: E231 missing whitespace after ','
scapy/layers/l2.py:435:42: E231 missing whitespace after ','
scapy/layers/l2.py:436:37: E231 missing whitespace after ','
scapy/layers/l2.py:436:39: E231 missing whitespace after ','
```

File: scapy/config.py (hunk 1 above)
Comment (pvinci) [reply scapy/config.py]: How should it be loaded?

Comment (gpotter2) [reply scapy/config.py]: Either `load_contrib("geneve")` or `from scapy.contrib.geneve import *`

**Pull request merged**